docs/source/en/_toctree.yml

```diff
@@ -43,25 +46,12 @@
     title: DreamBooth
   - local: using-diffusers/textual_inversion_inference
     title: Textual inversion
-  title: Adapters
+
+- title: Inference
   isExpanded: false
-- sections:
-  - local: using-diffusers/unconditional_image_generation
-    title: Unconditional image generation
-  - local: using-diffusers/conditional_image_generation
-    title: Text-to-image
-  - local: using-diffusers/img2img
-    title: Image-to-image
-  - local: using-diffusers/inpaint
-    title: Inpainting
-  - local: using-diffusers/text-img2vid
-    title: Video generation
-  - local: using-diffusers/depth2img
-    title: Depth-to-image
-  title: Generative tasks
```

Comment:
Where are these going?

Reply:
These are included in a "Task recipes" [section](https://moon-ci-docs.huggingface.co/docs/diffusers/pr_11936/en/using-diffusers/unconditional_image_generation) within "Resources"